### PR TITLE
more server work

### LINKED ISSUE #8: Upload Transcript Endpoint

### LINKED ISSUE #12: User and User Reviews Endpoint(s)
GET /user/<user_id>

Response: {
Email,
Reviews : [{
ReviewID,
CourseReview: { text, liked, useful, easiness}
InstructorReview: { text, liked, clear, engaging }
}, {....}],
Courses: [{
Id,
Subject_code,
Number,
Title
}, {...} ]
}

### LINKED ISSUE #18: Review Endpoint(s)

### LINKED ISSUE #47: Delete Review Endpoint

### PR DESCRIPTION
**Pull request by aelgohar**
more api work 

@raysarinas @MXSH-Dev will adjust this as you send me the requests from the client, but idk what they look like rn 

closes #12 
closes #47 
closes #18 
closes #8

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -118,40 +114,104 @@ def search():
...
+        )
+        .first_or_404()
+    )
+
+    course_review = CourseReview(
+        course_id=course.id,
+        review_text=lorem.sentence(),
+        liked=3,
+        easiness=4,
+        usefulness=2,
```
Comment (raysarinas) [inline server/app.py:190]: are these hardcoded?

File: server/app.py [hunk 2]
```diff
@@ -118,40 +114,104 @@ def search():
...
+    return 200
 
 
 @app.route("/user/<user_id>/review/<review_id>")
 def delete_review(user_id, review_id):
-    pass
+    user = User.query.filter(User.id == user_id).first_or_404()
+    review = Review.query.filter(Review.id == review_id).first_or_404()
+
+    if review.user_id != user.id:
```
Comment (raysarinas) [inline server/app.py:209]: can u add a TODO here for also allowing admins to delete reviews?

**Review by raysarinas — CHANGES_REQUESTED** [review]

File: server/app.py (hunk 1 above)
Comment (MXSH-Dev) [reply server/app.py:190]: I guess it's ok for now for testing purposes, will change later

File: server/app.py (hunk 2 above)
Comment (MXSH-Dev) [reply server/app.py:209]: OK, I added a task for this in github projects, and will merge this pull request

**Pull request merged**